pkg/pcr/flow.go

```diff
@@ -55,6 +63,12 @@ const (
 
 	// FlowIntelLegacyTXTEnabledTPM12 means a pre-CBnT flow with enabled TXT for TPM 1.2
 	FlowIntelLegacyTXTEnabledTPM12
+
+	// FlowLegacyPSPDisabled means an AMD flow with disabled PSP and with initial set of measurements
+	FlowLegacyPSPDisabled
+
+	// FlowLegacyPSPEnabled means an AMD flow with enabled PSP and with initial set of measurements
+	FlowLegacyPSPEnabled
```

Comment:
Why `Legacy`? In Intel we just used `Legacy` to emphasize that `TXT` of **pre**-CBnT flow.

Reply:
Good question, because it the initial implementation of the measurements.
More measurements are expected to appear.

Reply:
OK, makes sense.